code/game/objects/structures/crates_lockers/crates/large.dm

```diff
@@ -11,18 +11,18 @@
 	close_sound = 'sound/machines/wooden_closet_close.ogg'
 	open_sound_volume = 25
 	close_sound_volume = 50
+	speciallock = TRUE
 
 /obj/structure/closet/crate/large/attack_hand(mob/user)
 	add_fingerprint(user)
 	if(manifest)
-		tear_manifest(user)
+		tear_manifest(FALSE, FALSE, user)

```

Comment:
The entire process of calling this from here is questionable. Either send src as the first argument or send null. At least try to send the correct types down.